### PR TITLE
Add new plugins to What's New sections (e111)

### PR DESCRIPTION
**Pull request by nakib103**
sandbox - http://wp-np2-11.ebi.ac.uk:7070/info/docs/tools/vep/script/vep_download.html#new

### CODE REVIEW

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html [hunk 1]
```diff
@@ -133,6 +133,8 @@ <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_R
 
     <li>New plugins (supported on CLI): </li>
       <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/DeNovo.pm" rel="external">DeNovo</a> - identifies de novo variants in a VCF file</li>
```
Comment (jamie-m-a) [inline docs/htdocs/info/docs/tools/vep/script/vep_download.html:136]: Can you add to the end: requires a pedigree (.ped) file

**Review by jamie-m-a — CHANGES_REQUESTED** [review]

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html (hunk 1 above)
Comment (nakib103) [reply docs/htdocs/info/docs/tools/vep/script/vep_download.html]: @jamie-m-a , added!

File: docs/htdocs/info/docs/tools/vep/script/vep_download.html [hunk 2]
```diff
@@ -133,6 +133,8 @@ <h4>New in version [[SPECIESDEFS::ENSEMBL_VERSION]] <i>([[SPECIESDEFS::ENSEMBL_R
 
     <li>New plugins (supported on CLI): </li>
       <ul>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/DeNovo.pm" rel="external">DeNovo</a> - identifies de novo variants in a VCF file. This plugin requires a pedigree (.ped) file.</li>
+        <li><a href="//github.com/Ensembl/VEP_plugins/blob/release/[[SPECIESDEFS::ENSEMBL_VERSION]]/SpliceVault.pm" rel="external">SpliceVault</a> - predict exon-skipping events and activated cryptic splice sites based on the most common mis-splicing events around a splice site</li>
```
Comment (azangru) [inline docs/htdocs/info/docs/tools/vep/script/vep_download.html:137]: `predict` -> `predicts`? Similar to the 3rd-person verbs beginning the other lines?

Comment (jamie-m-a) [reply docs/htdocs/info/docs/tools/vep/script/vep_download.html:137]: Could do for consistency I guess

**Review by jamie-m-a — APPROVED** [review]
Fine from my side, btu could make the minor change Andrey suggests

**Comment by nakib103** [comment]
Thanks @jamie-m-a @azangru , updated accordingly!

**Pull request merged**